interop_client/src/main.rs

```diff
@@ -174,6 +174,7 @@ impl MlsClient for MlsClientImpl {
         request: Request<SupportedCiphersuitesRequest>,
     ) -> Result<Response<SupportedCiphersuitesResponse>, Status> {
         let request = request.get_ref();
+        info!(?request, "Request");
```

Comment:
Can you instead remove the line above and make the argument a `_request`? This request is always empty so there's nothing to log here.